lib/evss/letters/configuration.rb

```diff
@@ -2,7 +2,12 @@
 module EVSS
   module Letters
     class Configuration < EVSS::Configuration
-      DEFAULT_TIMEOUT = Settings.evss.letters.timeout
+      def request_options
+        {
+          open_timeout: open_timeout,
```

Comment:
where does the value `open_timeout` come from?

Reply:
https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/common/client/configuration/base.rb#L14